client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -58,6 +58,7 @@ import {
 	getPlanSlug,
 } from 'calypso/state/plans/selectors';
 import getCurrentPlanPurchaseId from 'calypso/state/selectors/get-current-plan-purchase-id';
+import { useSiteGlobalStylesStatus } from 'calypso/state/sites/hooks/use-site-global-styles-status';
```

Comment:
We are trying to decouple anything under `plan-features-2023-grid` from Calypso state and other dependencies, and this is just adding more to that work. Can you rework this to pass the value down from `plans-features-main` instead?

Reply:
and I just realised we haven't merged the change yet (typed Async props). It's in https://github.com/Automattic/wp-calypso/pull/78731

Reply:
@chriskmnds this should be fixed with dfb3ee0dad9bd24057a801f00d29979582f3e13a, but I'm a bit lost in the flow of the code and all the potential testing venues.
Would you mind confirming it's working as intended when you get the chance?

Reply:
Seems to be working for me, thanks @Copons!

Reply:
Thank you. All good from my end. also tested a flow and admin and works as outlined.